honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,327 @@
+                    'honyebee generic air wall modifier.'
+    )
+
+
+class BaseModifierSet(NoExtraBaseModel):
+    """Base class for the modifier sets assigned to Faces."""
+
+    exterior_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A radiance modifier object for faces with an Outdoors boundary '
+                        'condition.'
+        )
+
+    interior_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A radiance modifier object for faces with a boundary condition '
+                        'other than Outdoors.'
+        )
+
+
+class WallModifierSet(BaseModifierSet):
+    """Set containing radiance modifiers needed for a model's Walls."""
+
+    type: constr(regex='^WallModifierSet$') = 'WallModifierSet'
+
+
+class FloorModifierSet(BaseModifierSet):
+    """Set containing radiance modifiers needed for a model's Floors."""
+
+    type: constr(regex='^FloorModifierSet$') = 'FloorModifierSet'
+
+
+class RoofCeilingModifierSet(BaseModifierSet):
+    """Set containing radiance modifiers needed for a model's roofs."""
+
+    type: constr(regex='^RoofCeilingModifierSet$') = 'RoofCeilingModifierSet'
+
+
+class ShadeModifierSet(BaseModifierSet):
+    """Set containing radiance modifiers needed for a model's Shade."""
+
+    type: constr(regex='^ShadeModifierSet$') = 'ShadeModifierSet'
+
+
+class ApertureModifierSet(NoExtraBaseModel):
+    """Set containing radiance modifiers needed for a model's Apertures."""
+
+    type: constr(regex='^ApertureModifierSet$') = 'ApertureModifierSet'
+
+    window_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, '
+                        'and Wall parent Face.'
+        )
+
+    interior_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A modifier object for apertures with a Surface '
+                        'boundary condition.'
+        )
+
+    skylight_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, and a '
+                        'RoofCeiling or Floor face type for their parent face.'
+        )
+
+    operable_modifier: _REFERENCE_UNION_MODIFIERS = Field(
+            default=None,
+            description='A modifier object for apertures with an Outdoors boundary '
+                        'condition and a True is_operable property.'
+        )
+
+
+class DoorModifierSet(BaseModifierSet):
+    """Set containing radiance modifiers needed for a model's Doors."""
+
+    type: constr(regex='^DoorModifierSet$') = 'DoorModifierSet'
+
+    exterior_modifier: _REFERENCE_UNION_MODIFIERS = Field(
```

Comment:
This `exterior_modifier` is the same as the `BaseModifierSet`. So you can delete it.